decidim-core/spec/services/decidim/notifications_subscriptions_persistor_spec.rb

```diff
@@ -43,7 +43,7 @@ module Decidim
 
     describe "#delete_subscription" do
       let(:user) do
-        create(:user, organization:, "notification_settings" => {
+        create(:user, :organization => organization, "notification_settings" => {
```

Comment:
The syntax here could converted to the new syntax.

Reply:
Now it's on 984ace2000ad97e768bb8eba091d4ba7eda2ac11 - I had to force push because I missed a change